backend/fms_core/models/sample.py

```diff
@@ -167,6 +168,8 @@ class Sample(TrackedModel):
     child_of = models.ManyToManyField("self", blank=True, through="SampleLineage",
                                       symmetrical=False, related_name="parent_of")
 
+    projects = models.ManyToManyField("project", blank=True, through="SampleByProject", related_name="samples")
```

Comment:
I don't think you need to create ManyToMany relationship on both end of the relationship. One side should be enough to provide the same result (either on the sample or the project)... The related_name will point to the other side of the relationship...

Reply:
So on one of them it would have to be just foreign key?